package/crds/null.template.jet.crossplane.io_resources.yaml

```diff
@@ -1,8 +1,7 @@
-apiVersion: apiextensions.k8s.io/v1
```

Comment:
`crds.clean` target should either remove only a single line or the target to be removed completely. We're still [figuring that out](https://crossplane.slack.com/archives/CEF5N8X08/p1645605213253989) but it's not mergeable right now without this line.